pages/linux/tcptraceroute.md

```diff
@@ -0,0 +1,28 @@
+# tcptraceroute
+
+> A traceroute implementation using TCP packets.
+> More information: <https://github.com/mct/tcptraceroute>.
+
+- Trace the route to a host:
+
+`tcptraceroute {{host}}`
+
+- Specify the destination port and packet length in bytes:
+
+`tcptraceroute {{host}} {{destination_port}} {{packet_length}}`
+
+- Specify the local source port and source address:
+
+`tcptraceroute {{host}} -p {{source_port}} -s {{source_address}}`
+
+- Set the initial and maximum TTL:
```

Comment:
Let's specify the unit also.

EDIT: Here the description mentions initial, but the flag mentions "first". We should be consistent and use the same thing everywhere. I am leaning towards using first in the description.

Reply:
Oops! I didn't catch this.

Reply:
I was following the man pages. If that's not preferred, then I will change line 24 as well: 

- Specify the wait time and number of ~probes for each~ queries per hop:

Reply:
Yeah, we usually want to simplify the language and make it consistent with our overall theme and tone. If we just copy from the man page, then there's no point. :)

Your new change sounds good as well.